draft-ietf-tls-tls13.md

```diff
@@ -2958,7 +2957,16 @@ the responses).
 
 ### Key and IV Update {#key-update}
 
-     struct {} KeyUpdate;
+     enum { true(1), false(2) (255) } Boolean;
```

Comment:
You don't need the (255), yes, but it is at least somewhat conventional to have it just to remind readers of the width of the length field.